packages/gradient-block/package.json

```diff
@@ -0,0 +1,29 @@
+{
+	"name": "gradient-block",
+	"version": "1.0.0",
+	"main": "src/index.ts",
+	"scripts": {
+		"serve": "frontify-cli serve --entryPath src/index.ts",
+		"deploy": "frontify-cli deploy --entryPath src/index.ts",
+		"lint": "eslint .",
+		"lint:fix": "eslint . --fix",
+		"typecheck": "tsc --noEmit"
+	},
+	"dependencies": {
+		"@frontify/app-bridge": "3.0.0-beta.38",
+		"react": "^17.0.2",
+		"react-dom": "^17.0.2"
+	},
+	"devDependencies": {
+		"@frontify/eslint-config-react": "0.15.5",
+		"@frontify/frontify-cli": "5.3.6",
+		"@frontify/guideline-blocks-settings": "0.25.10",
+		"@types/react": "17.0.53",
+		"@types/react-dom": "17.0.18",
+		"autoprefixer": "10.4.13",
+		"eslint": "8.33.0",
+		"prettier": "2.8.3",
+		"tailwind": "4.0.0",
+		"typescript": "4.9.4"
+	}
+}
```

Comment:
Please add missing EOF (posix ending)

Reply:
Cool, thanks for pointing this out! Will do!

Then I guess it also makes sense to update the `npx @frontify/frontify-cli@latest create` script that generates the file in the first place ;)